javascript/parseXmlLatex.js

```diff
@@ -159,6 +164,14 @@ const processTextFunctionsDefaultLatex = {
     writeTo.push("}");
   },
 
+  META: (node, writeTo) => {
+    writeTo.push("$\\mathit{");
+    let s = node.firstChild.nodeValue;
+    s = s.replace(/-/g, "\\mhyphen ");
```

Comment:
Maybe  `\\mhyphen{}` instead of the extra space? That is probably more robust.

Reply:
Done.